content/chapter/03.tex

```diff
@@ -197,7 +197,18 @@ \subsubsection*{Messung der Unwucht}
-Es existiert eine weitere Möglichkeit den Sensor auf eine höhere Messrate zu bekommen, welche im Datenblatt jedoch nur sehr kurz und knapp erklärt ist. Hierbei ist es möglich das Gyroskop des Sensor abzuschalten und den Beschleunigungssensor mittels eines "Burst-Modus" (multiple reads) auszulesen. Den Sensor in diesen Modus zu setzen und auszulesen war allerdings aufgrund Mangel des technischen Verständnisses und der Datenblatterklärung nicht möglich.
-Es kann davon ausgegangen werden, dass die benötigte Zeit für die Messung der Sensoren bei dem eingesetzten Sensor schon intern zu lange dauert. Die Empfehlung (für zukünftige Projekte ?) liegt daher auf dem Einsatz von anderen Sensoren, welche für diesen Einsatzzweck besser geeignet sind. Ein Beispiel hierfür ist der Bosch BMA180. Dies ist ein dreiachsichger Beschleunigungssensor, welcher intern mit einer Messgeschwindigkeit von 1200Hz operieren kann. Damit kommt dieser Sensor auf ein komplett neues Messergebniss alle 417$\mu$s (Quelle: http://irtfweb.ifa.hawaii.edu/~tcs3/jumpman/jumppc/1107-BMA180/BMA180-DataSheet-v2.5.pdf). Hiermit sind bei einer Umdrehungszeit von ~20ms bis zu 48 Messergebnisse möglich, welche wahrscheinlich ausreichend sind für eine Aussage über die Unwuchtlokalisierung. Eine weitere Möglichkeit für den Ersatz des Sensor wäre der Einsatz einer inertialen Messeinheit (englisch \textit{inertial measurement unit}, IMU). Dies ist meist eine räumliche Kombination von mehreren Sensoren, welche auch Einsatz in heutigen Drohnen finden. Ob sich diese für den hier benötigten Einsatz verwenden liesen ist nicht bekannt und müsste weiter recherchiert werden.
+Zusammenfassend lassen sich die Probleme der Detektion und Lokalisierung der Unwucht auf folgenden Punkt vereinheitlichen: das Auslesen der Messwerte des Beschleunigungssensors benötigt zu viel Zeit.
+Es ist lediglich möglich, drei bis vier Messungen pro Umdrehung (circa 24 \ac{ms}) des Propellers zu erhalten.
+Dadurch existieren nicht genügend Werte, welche für eine aussagekräftige Analyse notwendig sind.
+
+Ein Versuch dieses Problem zu lösen, also die Messgeschwindigkeit des Sensors zu erhöhen, bestand darin, die Verbindungsgeschwindigkeit zwischen \ac{A2} und \ac{9-DOF} zu erhöhen.
+Die Verbindung zwischen den beiden Geräten findet über die Schnittstelle \ac{I2C} im normalen Modus statt.
+Durch Manipulation der \emph{twi.h}, eine Header-Datei der Arduino-Bibliothek \emph{Wire.h}, welche die Verbindungsparameter für die \ac{I2C}-Schnittstelle steuert, war es möglich die Verbindung im \emph{Fast-Mode} zu betreiben.
```

Comment:
Wie hast du sie manipuliert? 
Und pack den manipulierten Quellcode doch in den Anhang